app/routes/groups_.$groupId.tsx

```diff
@@ -1,10 +1,11 @@
 import type { LoaderFunction } from '@remix-run/node';
 import { useParams, useLoaderData, Link } from '@remix-run/react';
 import { db } from '~/modules/database/db.server';
+import { EventCard } from '~/components/ui/eventCard';
 import { json } from '@remix-run/node';
 
 export const loader: LoaderFunction = async ({ params }) => {
-  const group = await db.group.findFirstOrThrow({ where: { id: params.groupId } });
+  const group = await db.group.findFirstOrThrow({ where: { id: params.groupId }, include: { events: true } });
```

Comment:
Because you mentioned this in the PR description: `Event` and `Group` are two independent tables in our relational database. By including events in a group query, we `join` the two tables together to get the associated events for each group. By default, Prisma retrieves all fields of the `Group` table but not also the fields of the event table (the event object) because that would be very costly performance-wise to do this by default for every query, even if you don't need the events data!